test/selenium-test/selenium/okta-angular-widget-test.ts

```diff
@@ -0,0 +1,54 @@
+var webdriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+var chromeCapabilities = webdriver.Capabilities.chrome();
+
+var browser = new webdriver.Builder().forBrowser('chrome')
+                                         .setChromeOptions(new chrome.Options().addArguments(
+                                              "--headless",
+                                              "--disable-dev-shm-usage",
+                                              "--verbose",
+                                              "--disable-web-security",
+                                              "--ignore-certificate-errors",
+                                              "--allow-running-insecure-content",
+                                              "--allow-insecure-localhost",
+                                              "--no-sandbox",
+                                              "--disable-gpu"
+                                         ))
+                                         .build();
+var assert = require('assert');
+(async function example() {
```

Comment:
Why is this function named `example`? Name it something meaningful like `e2eTest`?

Reply:
@vijetmahabaleshwar-okta 
Done!